src/entities/friendships.rs

```diff
@@ -292,3 +292,13 @@ impl FriendshipRepositoryImplementation for FriendshipsRepository {
         }
     }
 }
+
```

Comment:
I would add some doc here. Sth like:

"/// takes in a tuple of two string references and returns the same tuple, sorted in ascending order based on their lexicographic order. If `address1` is less than `address2`, the original tuple addresses is returned as is. Otherwise, the tuple (address2, address1) is returned."

Reply:
I commented anywhere haha. I mean the `order_addresses` function